package.json

```diff
@@ -18,10 +22,18 @@
     "webgme": "^2.23.0"
   },
   "devDependencies": {
+    "@types/node": "^18.7.18",
+    "@types/underscore": "^1.11.4",
     "benchmark": "^2.1.4",
     "chai": "^4.3.4",
     "commander": "^9.4.0",
     "mocha": "^5.2.0",
+    "typescript": "^4.8.3",
+    "vite": "^3.1.4",
+    "vite-plugin-dts": "^1.6.1",
     "webgme": "^2.42.1"
+  },
+  "dependencies": {
+    "changeset": "github:deepforge-dev/changeset"
   }
```

Comment:
This should be a dev dependency since it will be bundled with WJI in the vite build